app/assets/stylesheets/pokeys.scss

```diff
@@ -0,0 +1,91 @@
+// Place all the styles related to the Pokeys controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+
+/* Navbar **************************************************************************************/
+$primary-color: #333;
+
+nav {
+  ul {
```

Comment:
try this
`nav ul {`...

You can simplify these by using https://www.w3.org/TR/css3-selectors/#selectors